fem/bilinearform_ext.cpp

```diff
@@ -42,9 +42,12 @@ PABilinearFormExtension::PABilinearFormExtension(BilinearForm *form)
      trialFes(a->FESpace()),
      testFes(a->FESpace())
 {
-   ElementDofOrdering ordering = UsesTensorBasis(*a->FESpace())?
-                                 ElementDofOrdering::LEXICOGRAPHIC : ElementDofOrdering::NATIVE;
-   elem_restrict_lex = trialFes->GetElementRestriction(ordering);
+   const Operator* elem_restrict =
+      trialFes->GetElementRestriction(UsesTensorBasis(*a->FESpace())?
+                                      ElementDofOrdering::LEXICOGRAPHIC :
+                                      ElementDofOrdering::NATIVE);
+
+   elem_restrict_lex = dynamic_cast<const ElementRestriction*>(elem_restrict);
    if (elem_restrict_lex)
```

Comment:
@YohannDudouit, can you confirm that I resolved the merge conflicts correctly here?

Reply:
I think we should rename `elem_restrict_lex` to `elem_restrict` since it can now be `NATIVE` too, this was my fault, but this can be done later.
The `dynamic_cast` is going to always fail if the restriction is an `L2ElementRestriction`, which I now realize is probably wrong, and will also cause trouble with DG-PA. So I think we should leave `elem_restrict_lex` type to `Operator*`, and move this `dynamic_cast` just before the call to `elem_restrict_lex->MultTransposeUnsigned(localY, y)`.
I can do that when merging master with DG-PA if you prefer to move on with this PR.

Reply:
I did the renaming of `elem_restrict_lex` in the DG PA branch, no need to do it here.